core/commands/configure_command.rb

```diff
@@ -123,34 +121,24 @@ def input_mdbe_settings
     { 'key' => read_topic('Please input the private key for MariaDB Enterprise', @configuration.dig('mdbe', 'key')) }
   end
 
-  def input_or_create_security_group(credentials)
-    if read_topic('Create new AWS security group?', 'y').casecmp('y').zero?
-      aws_service = AwsService.new(credentials, @ui)
-      security_group = aws_service.create_security_group
```

Comment:
The corresponding method of the AWSService class is no longer used anywhere. Please remove it.